### PR TITLE
Prashant_Assignment_8_Done

### PR DESCRIPTION
**Pull request by ghost**
Prashant_Assignment_8_Done. Please check and review.

### CODE REVIEW

File: src/prashantCJune20/Bank.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package prashantCJune20;
+public class Bank {
+	
+	
+	int creditCount = 0;
```
Comment (KrishnaKTechnocredits) [inline src/prashantCJune20/Bank.java:5]: Remove the initialization of all instance variables.

File: src/prashantCJune20/Bank.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	
+	void userDetails (String userName,int userbalance){
+		balance = userbalance;
+		name = userName;
+		
+	}
+	
+	
+	void debit(int amount ) {
+		if (amount > 0)
```
Comment (KrishnaKTechnocredits) [inline src/prashantCJune20/Bank.java:24]: You should write the condition to verify the insufficient balance.

File: src/prashantCJune20/Bank.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+		{
+			balance = balance + amount;
```
Comment (KrishnaKTechnocredits) [inline src/prashantCJune20/Bank.java:26]: Bhai debit credit ka code ulata likh diya hai, debit me kab se amount add hone lagi

File: src/prashantCJune20/Bank.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+		}
+		else{
+		System.out.println("Not provided any amount to credit");	
+		}
+					
+	}
+	void credit(int cr){
+		
+		if(cr <= balance){
+			balance = balance-cr;
```
Comment (KrishnaKTechnocredits) [inline src/prashantCJune20/Bank.java:39]: Credit me kab se amount delete hone lagi

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Take care of comments.

File: src/prashantCJune20/Bank.java (hunk 4 above)
Comment (ghost) [reply src/prashantCJune20/Bank.java:39]: aare haa sorry bhabhi wo raat mai 2 baje kar raha tha.. So sorry i will take care next time.

**Pull request merged**